rookout/manifest.json

```diff
@@ -1,28 +1,57 @@
 {
-  "integration_id": "rookout",
-  "name": "rookout",
-  "manifest_version": "1.0.0",
-  "maintainer": "support@rookout.com",
-  "display_name": "Rookout",
-  "short_description": "Production debugging, logging and alerting",
-  "creates_events": true,
-  "public_title": "Datadog-Rookout Integration",
-  "supported_os": [
-    "linux",
-    "mac_os",
-    "windows"
+  "manifest_version": "2.0.0",
+  "app_uuid": "a82a4f89-0690-48cf-bad0-9603fb652f44",
+  "app_id": "rookout",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Supported OS::Mac OS",
+    "Supported OS::Windows"
   ],
-  "support": "contrib",
-  "guid": "ad342dd9-4fe8-44e6-8bee-1e1cc64b1d28",
-  "type": "crawler",
-  "categories": [
-    "exceptions"
-  ],
-  "is_public": true,
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
```

Comment:
```suggestion
```

Lets either remove this or add an H2 "Support" section in the README.

Reply:
Done